packages/union-component/src/machines/__tests__/membershipMachine.spec.ts

```diff
@@ -16,7 +16,8 @@ test('goes into process membership state after filling all information', () => {
     lastName: faker.name.lastName(),
     email: faker.internet.email('bot', '', 'debtcollective.org'),
     phoneNumber: faker.phone.phoneNumber('+# ### ### ####'),
-    chapter: 'massachusetts'
+    chapter: 'massachusetts',
+    fundId: '1'
```

Comment:
as the `fn` signatures I see on the implementation this seems to be a number.
_Somethig needs to be updated: the fn works with string? or the value is numbers? because I see "casting" on `funds[0].id as string`_